apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/style.scss

```
@@ -74,69 +72,46 @@ $transition-period: 200ms;
 	top: 0;
 	left: 0;
 	height: $header-height !important;
-	animation: wpcom-block-editor-nav-sidebar-nav-sidebar__shrink $transition-period ease-in-out forwards;
-	@include reduce-motion( 'animation' );
-
-	&.is-growing {
-		animation: wpcom-block-editor-nav-sidebar-nav-sidebar__shrink $transition-period ease-in-out reverse;
-		@include reduce-motion( 'animation' );
-	}
-}
-
-@keyframes wpcom-block-editor-nav-sidebar-nav-sidebar__shrink {
-	0% {
-		transform: scale( 1 );
-	}
-	100% {
-		transform: scale( 0.55 );
-	}
-}
-
-.wpcom-block-editor-nav-sidebar-nav-sidebar__home-button-container {
-	border-bottom: $border;
 }
 
 .wpcom-block-editor-nav-sidebar-nav-sidebar__home-button {
 	height: 46px;
 	width: 100%;
-	font-weight: 600;
 	border: none !important;
 	box-shadow: none !important;
+	color: $white;
+	flex-shrink: 0;
+	padding: 8px 11px !important;
+	margin: 14px 0;
 
 	&:hover {
-		text-decoration: underline;
+		color: $white;
+	}
+
+	&:focus {
+		color: $white;
+		border: 1px solid $white !important;
+		box-shadow: none !important;
+		outline: none !important;
+		border-radius: 0;
+		padding: 7px 10px !important;
 	}
 }
 
+.wpcom-block-editor-nav-sidebar-nav-sidebar__list-heading {
+	padding: $panel-padding;
+	font-size: $big-font-size;
+	color: $white;
+}
+
 .wpcom-block-editor-nav-sidebar-nav-sidebar__page-list {
 	padding: 0;
 	margin: 0;
 	overflow-y: scroll;
```

This should be changed to `auto` to prevent scrollbar appearing unnecessarily
```suggestion
	overflow-y: auto;
```

---

Oh that's gross ... trying to think why I'm not seeing that in the browsers I tested. But I think you're right, it's probably going to be gross on Windows browsers where the scrollbar always takes up space.